SU-HSDO/suhumsci
# fix(shs-5118): add to calendar theming

# READY FOR REVIEW

## Summary
This PR adds theming adjustments to the Add to Calendar component, based on [Figma designs](https://www.figma.com/proto/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?type=design&node-id=640-102&t=LhLJzFZTEJQDsIJn-0&scaling=min-zoom&page-id=635%3A20&hide-ui=1).

## Urgency
medium

## Steps to Test
1. Visit a page like https://dlcl.stanford.edu/events/french-speaking-worlds-rejane-senac (local copy: https://dlcl.suhumsci.loc/events/french-speaking-worlds-rejane-senac) 
2. Verify that the theming adjustments match those in the [Figma design.](https://www.figma.com/proto/hOtD5MStjFIavUrWapHKHN/UX%2FDesign-Tickets-Work?type=design&node-id=640-102&t=LhLJzFZTEJQDsIJn-0&scaling=min-zoom&page-id=635%3A20&hide-ui=1)  

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
